app/stats/status.rb

```diff
@@ -1,23 +1,79 @@
-require_relative "./level.rb"
-require_relative "./force.rb"
-require_relative "./defense.rb"
-require_relative "./hit_point.rb"
-require_relative "./magic_point.rb"
+require_relative "stats.rb"
+require_relative "hit_point.rb"
+require_relative "magic_point.rb"
+require_relative "force.rb"
+require_relative "defense.rb"
+require_relative "dexterity.rb"
+require_relative "luck.rb"
+
 
 module Stats
 
     class Status
-        attr_accessor :hit_point, :magic_point, :force, :defense
```

Comment:
Now you can define globale variables dynamically : 

``ruby
        def initialize(args=nil)

          ["hit_point", "magic_point", "force", "dexterity", "defense", "luck"].each do |stat|
            value = if args && args["#{stat}".to_sym]
              args["#{stat}".to_sym]
            else
              Object.const_get "Stats::DEFAULT_#{stat.upcase}"
            end

            instance_variable_set("@#{stat}", value)
          end
        end

```